balanced-parentheses/balanced-parentheses-helper.js

```diff
@@ -0,0 +1,28 @@
+/**
+ * Check if expression has balanced paranthesis.
+ * @param {string} expression The first number.
+ * @return {boolean} The sum of the two numbers.
+ */
+export const isBalanced = (expression) => {
+  console.log("EXPRESSION TO CHECK: ", expression);
+  if (!expression || expression.length == 0) return false;
+
+  const stack = [];
+  const ParanthesisMap = {
+    "(": ")",
+    "{": "}",
+    "[": "]",
+  };
+  const closeParantheses = [")", "}", "]"];
+
+  [...expression].every((item) => {
```

Comment:
why not use map here?

Reply:
map returns an array based on calling function result. But I need single value here